src/client/renderingengine.cpp

```diff
@@ -112,6 +114,8 @@ RenderingEngine::RenderingEngine(IEventReceiver *receiver)
 	driver = m_device->getVideoDriver();
 
 	s_singleton = this;
+
+	m_device->getGUIEnvironment()->setSkin(new gui::GUISkin(gui::EGST_WINDOWS_METALLIC, driver));
```

Comment:
this new pointer is dropped somewhere ?

Reply:
Checked docs, CGUIEnvironment handles the skin lifetime after this call. I was missing a ->drop(), however